pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -232,6 +232,8 @@
   tree-sitter-langs = callPackage ./tree-sitter-langs { final = self; };
   tsc = callPackage ./tsc { };
 
+  yes-no = callPackage ./yes-no { };
```

Comment:
EmacsWiki codes are basically unmaintained. Why should we include this?

Reply:
I'm still using this package, and it doesn't break in any newer version of Emacs (I'm using snapshot `29.0.50`).

From your concern, I have set myself as the maintainer. I would keep an eye on this package if it does somehow break in the future. On the other hand, we can always place the fixed (update to date) version somewhere then replace the current `emacsmirror` version.